diff --git a/libs/lottie.js b/libs/lottie.js
index 9ade8c1378595d3d06077fd9f701ed8a5837f0b3..8ca30b9488ee8f39233dec7dbf5a5be90e4d19fa 100644
--- a/libs/lottie.js
+++ b/libs/lottie.js
@@ -13059,7 +13059,7 @@ const lottie = (() => {
       var valueAtTime;
       var velocityAtTime;
       var scoped_bm_rt;
-      var expression_function = eval("[function _expression_function(){" + val + ";scoped_bm_rt=$bm_rt}]")[0];
+      var expression_function = (new Function('scoped_bm_rt', val + ';return $bm_rt;'))();
       var numKeys = property.kf ? data.k.length : 0;
       var active = !this.data || this.data.hd !== true;
       var wiggle = function wiggle2(freq, amp) {
